test/DGmethods/sphere/advection_sphere_lsrk.jl

```diff
@@ -0,0 +1,385 @@
+# Advection on the Sphere test case.
+#
+# This is a case similar to the one presented in:
+# @article{WILLIAMSON1992211,
+#   title = {A standard test set for numerical approximations to the
+#            shallow water equations in spherical geometry},
+#   journal = {Journal of Computational Physics},
+#   volume = {102},
+#   number = {1},
+#   pages = {211 - 224},
+#   year = {1992},
+#   doi = {10.1016/S0021-9991(05)80016-6},
+#   url = {https://doi.org/10.1016/S0021-9991(05)80016-6},
+# }
+#
+# This version runs the advection on the sphere as a stand alone test (no
+# dependence on CLIMA moist thermodynamics)
+#--------------------------------#
+#--------------------------------#
+# Can be run with:
+# Integration Testing: JULIA_CLIMA_INTEGRATION_TESTING=true mpirun -n 1 julia --project=@. advection_sphere_lsrk.jl
+# No Integration Testing: JULIA_CLIMA_INTEGRATION_TESTING=false mpirun -n 2 julia --project=@. advection_sphere_lsrk.jl
+#--------------------------------#
+#--------------------------------#
+# Can be run on the GPU with:
+# Integration Testing: JULIA_CLIMA_INTEGRATION_TESTING=true mpirun -n 1 julia --project=/home/fxgiraldo/CLIMA/env/gpu advection_sphere_lsrk.jl
+# No Integration Testing: JULIA_CLIMA_INTEGRATION_TESTING=false mpirun -n 1 julia --project=/home/fxgiraldo/CLIMA/env/gpu advection_sphere_lsrk.jl
+#--------------------------------#
+#--------------------------------#
+
+using MPI
+using CLIMA
+using CLIMA.Topologies
+using CLIMA.Grids
+using CLIMA.DGBalanceLawDiscretizations
+using CLIMA.DGBalanceLawDiscretizations.NumericalFluxes
+using CLIMA.MPIStateArrays
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.StrongStabilityPreservingRungeKuttaMethod
+using CLIMA.ODESolvers
+using CLIMA.GenericCallbacks
+using CLIMA.Vtk
+using LinearAlgebra
+using StaticArrays
+using Logging, Printf, Dates
+using Random
+
+@static if haspkg("CUDAnative")
+    using CUDAdrv
+    using CUDAnative
+    using CuArrays
+    const ArrayTypes = VERSION >= v"1.2-pre.25" ? (Array, CuArray) : (Array,)
```

Comment:
I've changed the other examples to only use `CuArray`s on GPU nodes (to avoid double testing)

Reply:
OK. Want me to modify this PR and stop bors or do new PR?

Reply:
Can be a new PR.

Reply:
Went ahead and fixed it since the CI timed out and I was sitting at the computer.